docs/DeveloperGuide.md

```diff
@@ -755,18 +755,55 @@ testers are expected to do more *exploratory* testing.
 
    1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts and todos. The window size may not be optimum.
 
-1. Saving window preferences
+2. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### Contact Management
+   
+#### Adding a contact
 
-### Deleting a contact
+1. Adding a new contact.
+   1. Prerequisite: There is no contact named `John Doe`.
+   2. Test case: `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
+   3. Expected: New contact with the same details added in the contact list.
+2. Adding a contact that has existed before.
+   1. Prerequisite: There is a contact named `Betsy Crowe` in the contact list.
+   2. Text case: `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567`
```

Comment:
Perhaps add fullstop at end of test cases for consistency? Or remove the full stops from some others.